herc/webservice.py

```diff
@@ -22,6 +22,9 @@ def initialize(self):
         self.set_header('Content-Type', 'application/json')
         self.set_header('Access-Control-Allow-Origin',	   '*')
         self.set_header('Access-Control-Allow-Credentials', 'true')
+        module_dir = os.path.dirname(__file__)
```

Comment:
Hmm. If we're going to do this, we should do it for all referenced data files, and do it something like [this question](http://stackoverflow.com/a/5423147/2941784).
@scottfrazer Thoughts?